builtin/providers/aws/resource_aws_vpc.go

```diff
@@ -57,29 +58,33 @@ func resourceAwsVpc() *schema.Resource {
 				Computed: true,
 			},
 
-			"tags": tagsSchema(),
+			//			"tags": tagsSchema(),
 		},
 	}
 }
 
 func resourceAwsVpcCreate(d *schema.ResourceData, meta interface{}) error {
-	ec2conn := meta.(*AWSClient).ec2conn
-
+	ec2conn := meta.(*AWSClient).awsEc2conn
+	cidr := d.Get("cidr_block").(string)
+	instance_tenancy := "default"
+	if v, ok := d.GetOk("instance_tenancy"); ok {
+		instance_tenancy = v.(string)
+	}
 	// Create the VPC
-	createOpts := &ec2.CreateVpc{
-		CidrBlock:       d.Get("cidr_block").(string),
-		InstanceTenancy: d.Get("instance_tenancy").(string),
+	createOpts := &ec2.CreateVPCRequest{
+		CIDRBlock:       &cidr,
+		InstanceTenancy: &instance_tenancy,
```

Comment:
Nitpick: go ahead and use `aws.String()` for these, instead of  `&` here. For `cidr`, you can just do this inline unless you need the `cidr` var elsewhere. 

    CIDRBlock:       aws.String(d.Get("cidr_block").(string))